feed/tests/test_comment_views.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from feed.models import Post, Comment
+from django.contrib.auth.models import User
+
+
+USERNAME1 = 'username1'
+USER_PASS = 'secret_pass'
+POST_TITLE = 'post title'
+POST_CONTENT = 'some post content'
+LOGIN_URL = '/login/'
+REDIRECT_URL_STATUS = 302
+PAGE_NOT_FOUND_STATUS = 404
+
+
+def new_comment_url(post_parent_id):
+    return f"/post/{post_parent_id}/comment/new/"
+
+
+def post_detail_view_url(post_id):
+    return f"/post/{post_id}/"
+
+
+@pytest.fixture
+def users(db):
+    user1 = User.objects.create_user(USERNAME1, password=USER_PASS)
+    return [user1]
+
+
+@pytest.fixture
+def post(db, users):
+    post = Post.posts.create(title=POST_TITLE, content=POST_CONTENT, author=users[0])
+    post.save()
+    return post
+
+
+@pytest.mark.django_db
+class TestCommentCreateView:
+    def test_create_new_comment_view_entrypoint_auth(self, users, post, client):
+        # Testing that for a logged in user, the new comment url gets a valid response
+        client.force_login(users[0])
+        response = client.get(new_comment_url(post.id))
+        assert response.status_code == 200
```

Comment:
consider asserting the template also please refer this
https://github.com/redhat-beyond/beyond-tutorial/blob/main/msgboard/tests.py

Reply:
Added template assertion